consensus/src/ordered_broadcast/types.rs

```diff
@@ -585,6 +585,8 @@ impl<P: Array, D: Digest> FixedSize for Ack<P, D> {
 #[derive(Clone, Debug, PartialEq)]
 pub enum Activity<C: Verifier, D: Digest> {
     /// A new proposal from a sequencer
+    ///
+    /// This activity is only emitted if the application has first verified the proposal.
```

Comment:
We don't verify our own proposals, right? So maybe also clarify that we won't see our own proposals emitted (if that's intended)

Reply:
(I do verify the encode/decode in the e2e tests but will do so explicitly)